content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,133 @@
+    directory for the cluster.
+
+1. Run the following commands on each cluster in the mesh where a virtual
+     machine connection should be established. See
+     [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
+     for more details on configuring an external CA.
+
+{{< text bash >}}
+$ cd samples/certs
+$ make vm
+$ kubectl create namespace istio-system
+$ kubectl create secret generic cacerts -n istio-system \
+    --from-file=vm/ca-cert.pem \
+    --from-file=vm/ca-key.pem \
+    --from-file=vm/root-cert.pem \
+    --from-file=vm/cert-chain.pem
+$ cd ../..
+{{< /text >}}
+
+1. Generate a `cluster.env` configuration file that informs the virtual machine
+   deployment which network CIDR to capture and redirect to the Kubernetes
+   cluster:
+
+    {{< text bash >}}
+    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ echo -n ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > $HOME/cluster.env
+    {{< /text >}}
+
+1. If the Kubernetes cluster makes workload requests to the virtual machine,
+   the ports used in the request must be specified in `cluster.env`. An
+   example of exposing the `3306` and `8080` ports on the virtual machine:
+
+    {{< text bash >}}
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> $HOME/cluster.env
+    {{< /text >}}
+
+1. Install Istio with mesh expansion features:
+
+    {{< text bash >}}
+    $ istioctl install -f install/kubernetes/operator/examples/vm/values-istio-meshexpansion.yaml
+    {{< /text >}}
+
+### Installing the virtual machine
+
+Run the following commands on the virtual machine you want to add to the Istio mesh:
+
+1. Copy the previously created `cluster.env` and `*.pem` files to the virtual machine.
+
+1. Install the Debian package with the Envoy sidecar.
+
+    {{< text bash >}}
+    $ curl -LO https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
+    {{< /text >}}
+
+1. Add an IP address for Istiod to `/etc/hosts`.  If using multiple networks,
+    replace `<ISTIOD_IP>`  with the ingress gateway service of istiod.
+    Revisit the instructions in the
+    [Determining the Ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports). If using a single network, substitute `<ISTIOD_IP>` with the service
+    IP address of the `istiod` service.
+
+    {{< text bash >}}
+    $ sudo echo "<ISTIOD_IP> istiod.istio-system.svc" >> /etc/hosts
+    {{< /text >}}
+
+    {{< idea >}}
+    A sophisticated option involves configuring DNS within the virtual
+    machine to reference an external DNS server. This option is beyond
+    the scope of this document.
+    {{< /idea >}}
+
+1. Obtain `tcp-istiod` port for Istio ingress gateway and add this to `cluster.env`. This
```

Comment:
can the istiod port be customized? Last I checked, this was hardcoded throughout the codebase. I don't understand why we have a conditional for obtaining this address throughout our documentation. The istiod port should always be 15012.

Reply:
for cases where users are using `node-port`

Reply:
@irisdingbj - I see. The various WG leads have decided somewhere else in a different thread, and I don't recall the PR, that node port is not a sustainable model and we should remove it from the documentation. As such, I'd be happy to hard code in this documentation.

For evals, people can always use metallb on whatever environment they are running.

Reply:
resolved. We are not supporting node port use cases unless a strong rationale arises.